test_scripts/Polices/build_options/007_ATF_P_TC_Policy_Table_Update_Trigger_After_N_Days_HTTP.lua

```diff
@@ -117,8 +117,8 @@ function Test:TestStep_Register_App_And_Check_That_PTU_Triggered()
   EXPECT_RESPONSE(CorIdRAI, { success = true, resultCode = "SUCCESS" })
 
   EXPECT_HMICALL("BasicCommunication.PolicyUpdate"):Times(0)
-  EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", { status="UPDATE_NEEDED" }, { status="UPDATING" }):Times(AtLeast(1))
-  EXPECT_NOTIFICATION("OnSystemRequest", { requestType = "HTTP" })
+  EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", { status="UPDATE_NEEDED" }, { status="UPDATING" }):Times(2)
+  EXPECT_NOTIFICATION("OnSystemRequest", { requestType = "LOCK_SCREEN_ICON_URL" }, { requestType = "HTTP" }):Times(2)
```

Comment:
@istoimenova, In this script order of OnSystemRequests are the following: "HTTP" and then "LOCK_SCREEN_ICON_URL"
Since there is no req-t on ordering of these requests please correct.